.github/workflows/Test.yml

```diff
@@ -25,18 +25,18 @@ jobs:
             # Without setting this, a failing test cancels all others
             fail-fast: false
             matrix:
-                julia-version: ["1.6", "1", "~1.10.0-0"]
+                julia-version: ["1.6", "1", "pre"]
```

Comment:
Maybe we can test on `["lts", "1", "pre"]`, unless we plan on being backwards compatible with 1.6 for longer than required.

Reply:
I added 1.6 because it was the LTS version.  I like the idea of testing on the most recent LTS version.